pkg/resource/manager.go

```diff
@@ -96,6 +97,7 @@ func NewResourceManager(ctx context.Context, resourceNames []string, wrapper api
 		} else if resourceName == config.ResourceNameIPAddressFromPrefix {
 			resourceProvider = prefix.NewIPv4PrefixProvider(ctrl.Log.WithName("ipv4 prefix provider"),
 				wrapper, workers, resourceConfig, conditions)
+			healthCheckers[ipv4ProviderHealthCheckSubpath] = resourceProvider.GetHealthChecker()
```

Comment:
Do you mean to use `ipv4PrefixProviderHealthCheckSubpath` ?

Reply:
Yes. Sorry about that. Thanks for catching it. Updated it now.